src/OpenTelemetry.Extensions.Enrichment/OpenTelemetry.Extensions.Enrichment.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion);$(NetMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry .NET SDK telemetry enrichment.</Description>
+    <NoWarn>$(NoWarn),CS1591</NoWarn>
+    <MinVerTagPrefix>Extensions.Enrichment-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
+    <EnableAnalysis>true</EnableAnalysis>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="7.0.2" />
+    <PackageReference Include="System.Memory" Version="4.5.5" />
+    <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestPrereleaseVersion)" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="$(OpenTelemetryCoreLatestPrereleaseVersion)" />
```

Comment:
could you add a comment why this requires Otel.Ext.Hosting package?

Reply:
Thanks for spotting this. Removed